google_images_download/google_images_download.py

```diff
@@ -279,11 +279,9 @@ def single_image(self,image_url):
         response.close()
 
         image_name = str(url).split('/')[-1].partition('?')[0]
-        if ".jpg" in image_name or ".gif" in image_name or ".png" in image_name or ".bmp" in image_name or ".svg" in image_name or ".webp" in image_name or ".ico" in image_name:
-            file_name = main_directory + "/" + image_name
-        else:
-            file_name = main_directory + "/" + image_name + ".jpg"
-            image_name = image_name + ".jpg"
+        if not image_name.lower().endswith(".jpg .gif .png .bmp .svg .webp .ico".split()):
+            image_name += ".jpg"
+         file_name = main_directory + "/" + image_name + ".jpg"
```

Comment:
I think  `file_name = os.path.join(main_directory, image_name)`  is correct and better. It uses the os modul and gets rid of the SECOND ".jpg".